### DISCUSSION TITLE
** Documentation **

### DISCUSSION THREAD

**Discussion by postmeback** [makeplane/plane#1401] [Q&A]
Hi,

I see the documentation is incomplete and flawed as well. I would like to know when the team will look into this.

P.S.: I would love to contribute to documenting it. I find this project pretty interesting.

**Comment by philipp-koch** [comment]
I had the same thought, exactly. Please let us know if there's something we can do to at least address the low-hanging fruits (e.g. typos at https://docs.plane.so/). :-) Because under "[contributing](https://docs.plane.so/#contributing)", the documentation website claims that there's an "Edit on GitHub" link in each the page footer -however, I don't see a footer at all...

**Comment by Chaitrali-Shinde** [comment]
I'm pretty new to the open source community and for a new bee like me, a good documentation is the most useful thing. If anyone can do documentation for beginners it will be helpful.
